### ISSUE TITLE
IMOS: check_local_time_zone - timezone can be fractional

### ISSUE THREAD

**Issue by lbesnard** [aodn/compliance-checker#11]
netcdf checker output:

```
    local_time_zone                    :3:     0/ 1 :  
        check_attributes               :3:     0/ 1 : Value is not in the
                                                      expected range

```

ncdump output:

```
local_time_zone = 9.5 ;
```

However 9.5 is a valid timezone (need to fix check_local_time_zone function)

Example with file:
/mnt/ebs/wip/ANMN/timeSeries/new_tb/IMOS_ANMN-NRS_CKOSTUZ_20120825T054709Z_NRSDAR_FV01_NRSDAR-1208-Surface-WQM-1_END-20121209T014808Z_C-20150324T034821Z.nc

**Comment by mhidas** [comment]
@fxmzb123 Another one for you, if you get to it.

**Comment by mhidas** [comment]
Fixed via #26
